### PR TITLE
Update salesforce connector release notes

### PR DESCRIPTION
**Pull request by alexandruatomei9**

### CODE REVIEW

File: modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -5,6 +5,29 @@ endif::[]
 
 Support Category: Select
 
+== 9.7.5
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc]: For the title, please use Salesforce Connector - Mule 4

File: modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc [hunk 2]
```diff
@@ -5,6 +5,29 @@ endif::[]
...
+
+*July 27, 2019*
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc]: Please change the date to *July 1, 2019*

Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc]: Sorry about that. Changed

File: modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/salesforce-connector-release-notes-mule-4.adoc]: done

**Pull request merged**